Shared/Shared/GameObject.h

```diff
@@ -52,5 +64,5 @@ class GameObject : public Serializable {
 	std::string material;
 	int animation;
 	int id;
-	int radius;
+	BoundingShape * boundingShape;
```

Comment:
Should we initialize this with nullptr?

Reply:
i call setBoundingShape(NULL) in all the GameObject constructors
[https://github.com/ucsd-cse-125-spring-2019/cse125-sp19-group5/blob/game-engine/Shared/Shared/GameObject.cpp#L13](url)